tests/unit/utils/test_fs.py

```diff
@@ -135,3 +136,11 @@ def test_get_parent_dirs_up_to(path1, path2, expected_dirs):
     result = get_parent_dirs_up_to(path1, path2)
 
     assert set(result) == set(expected_dirs)
+
+
+@pytest.mark.skipif(os.name != "nt", reason="Windows specific")
+def test_relpath_windows_different_drives():
+    path1 = os.path.join("A:", os.sep, "some", "path")
```

Comment:
let's use `ntpath` directly, so we don't even depend on running on windows. Note that you'll need to use different `sep` for that.

Reply:
I think if we will not skip it, we will miss the point of test:
-on linux, relpath will still behave posix-like, so in order to make it pass, we will need to do some patching, which will not actually test behaviour on `nt`, because instead of using system defaults there, we will patch them.

Reply:
@pared `ntpath` on linux gives the same error:
```
>>> ntpath.relpath("A:\ololo", "C:\")
  File "<stdin>", line 1
    ntpath.relpath("A:\ololo", "C:\")
                                    ^
SyntaxError: EOL while scanning string literal
>>> ntpath.relpath("A:\\ololo", "C:\\")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/ntpath.py", line 529, in relpath
    % (path_prefix, start_prefix))
ValueError: path is on drive A:, start on drive C:
```

Reply:
@pared sorry, didn't read it carefully enough. You are right, let's keep it this way. Thanks!

Reply:
True, but to make `relpath` work on linux we will need to
`patch.object(os, "path", ntpath)` and `patch.object(os, "name", "nt")` effectively overriding system defaults.